tests/commands/test_api_error.py

```diff
@@ -249,3 +249,60 @@ def test_record_tests(self):
 
         result = self.cli("record", "tests", "--session", self.session, "minitest", str(self.test_files_dir) + "/")
         self.assertEqual(result.exit_code, 0)
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {"LAUNCHABLE_TOKEN": CliTestCase.launchable_token})
```

Comment:
nits: If possible, can the test part `self.cli(...)` and `self.assertEqual(...)` be summarized below?

Reply:
Sorry I wanted to say that I want to divide into setup section and test section

```
# set up section
response.add(...) # verify
response.add(...) # record build
response.add(...) # subset
response.add(..) # subset

# test section

verify = self.cli("verify")
self.assertEqual(verify.exit_code, 0)

record_build = self.cli("record build --name xxx")
self.assertEqual(record_build.exit_code, 0)
...
```